### PR TITLE
doc: expand wpantund documentation for Thread

### PR DESCRIPTION
**Pull request by greg-fer**
Added a new page for Thread tools with sections about wpantund.
(KRKNWK-6807)

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

File: samples/openthread/ncp/README.rst [hunk 1]
```diff
@@ -67,7 +67,7 @@ User interface
 
 All the interactions with the application are handled using serial communication.
 
-For the interaction with the application, this sample uses `wpantund`_ process with wpanctl commands.
+For the interaction with the application, this sample uses :ref:`ug_thread_tools_wpantund` process with ``wpanctl`` commands.
```
Comment (edmont) [inline samples/openthread/ncp/README.rst:70]: Should we normally use monospaced ``wpanctl`` and ``wpantund`` along the docs? Why just here?

Comment (greg-fer) [reply samples/openthread/ncp/README.rst:70]: I thought about this, but these are names of the tools, aren't they? The monospaced formatting is to be used only for code within text.

File: doc/nrf/ug_thread_tools.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+wpantund
+========
+
+`wpantund`_ is a utility for providing a native IPv6 interface to a Network Co-Processor.
+When working with Thread, it is used for the interaction with the application by the following samples:
+
+* :ref:`ot_ncp_sample`
+
+The interaction is possible using commands proper of wpanctl, a module installed with wpantund.
```
Comment (kkasperczyk-no) [inline doc/nrf/ug_thread_tools.rst:25]: Did you mention writing "commands prompt of wpanctl" or "proper to wpanctl"? "Commands proper of" doesn't seems to be right to me.

File: samples/openthread/ncp/README.rst (hunk 1 above)
Comment (edmont) [reply samples/openthread/ncp/README.rst:70]: Actually they are both names and commands :)

Comment (greg-fer) [reply samples/openthread/ncp/README.rst:70]: But is ``wpanctl`` alone a command? For me, in the context of this sentence, it is a name.

Comment (edmont) [reply samples/openthread/ncp/README.rst:70]: Yeah, in this sentence it's a name.

File: doc/nrf/ug_thread_tools.rst (hunk 2 above)
Comment (greg-fer) [reply doc/nrf/ug_thread_tools.rst]: Good catch. Edited.

**Pull request merged**